kreuzberg/_ocr/_paddleocr.py

```diff
@@ -96,10 +97,95 @@ class PaddleOCRConfig:
     """Whether to recognize spaces."""
     use_zero_copy_run: bool = False
     """Whether to enable zero_copy_run for inference optimization."""
-
-
-class PaddleBackend(OCRBackend[PaddleOCRConfig]):
-    _paddle_ocr: ClassVar[Any] = None
+    gpu_memory_limit: float | None = None
```

Comment:
if we are doing this, we should define "__slots__", read up on this - it make instantiation faster. slots should be defined on the base class as well (OCRBackend) to be inherited correctly. 

You use slots when you defined class attributes that are dynamically initialized in the constructor.